jenkinsfile-examples/msbuild/Jenkinsfile

```diff
@@ -4,7 +4,7 @@ node {
 
 	stage 'Build'
 		bat 'nuget restore SolutionName.sln'
-		bat "\"${tool 'MSBuild'}\" SolutionName.sln /p:Configuration=Release /p:Platform=\"Any CPU\" /p:ProductVersion=1.0.0.${env.BUILD_NUMBER}"
+		bat '\"${tool \'MSBuild\'}\" SolutionName.sln /p:Configuration=Release /p:Platform=\"Any CPU\" /p:ProductVersion=1.0.0.${env.BUILD_NUMBER}'
```

Comment:
Won't work, this relies on Groovy interpolation for the `tool` step.

Reply:
I pushed a new version, could you tell me if this approach is the correct one?